gitlabform/gitlabform/core.py

```diff
@@ -61,15 +64,46 @@ def get(self, path, default=None):
 def configuration_to_safe_dict(method):
 
     @wraps(method)
-    def method_wrapper(self, project_and_group, configuration):
+    def method_wrapper(self, project_and_group, configuration, **kwargs):
 
-        return method(self, project_and_group, SafeDict(configuration))
+        return method(self, project_and_group, SafeDict(configuration), **kwargs)
 
     return method_wrapper
 
 
 class GitLabFormCore(object):
 
+    def log_diff(self, subject, given, current, only_changed=False, hide_entries=None):
```

Comment:
I see that you never call this method with non-default values of `only_changed` or `hide_entries`. Do you plan to add code that will use it soon? I am asking because if not then I suggest not adding code that will not be used.

Reply:
I find names `given` and `current` not very clear. I suggest changing them to `from_config` and `from_server`, like in a comment a few lines below.

Reply:
a. Yes, there are some sensitive variables which should be hidden. I played around with it, but didn't use for now. Same for only changed. It should be set to true by default, but for development/debugging it's better to have full picture.
b. `from_config` and `from_server` sounds better, indeed. I will fix it

Reply:
Hey @akaRem , can you please find the time to make those small fixes? I would really like to merge your PR. :)

Reply:
Ping @akaRem . :) PS It's a coincidence that exactly one more month has passed. It's NOT like I am doing this on a schedule. Really.

Reply:
Hey, @akaRem ! It would be cool if you could find the time to fix the minor things we discussed above. Especially as we have another person interested in this PR: https://github.com/egnyte/gitlabform/issues/61

Reply:
super friendly ping!

Reply:
Perhaps you could take this PR over, @glensc ? :)